### DISCUSSION TITLE
"Public" Notes

### DISCUSSION THREAD

**Discussion by Moonpile** [vigoren/foundryvtt-simple-calendar#370] [Ideas]
It would be nice if there were a way to mark a note "Public" so that any new players who join can automatically see any such notes. 

That said the latest version is a HUGE leap ahead. It looks fantastic!

**Comment by vigoren** [comment]
In the latest release, 2.1.27, an option to select all players has been added to the notes Who Can See field. This will make it so all current and future players will be able to see the note.
